config/locales/en.yml

```diff
@@ -7,6 +7,9 @@ en:
   service:
     name: Review criminal legal aid applications
 
+  urls:
+    eforms_url: 'https://portal.legalservices.gov.uk/oamfed/idp/initiatesso?providerid=eForms'
```

Comment:
I've seen your comment on the ticket, but yeah I think that this will be the link for providers. I don't think caseworkers use portal to access the services they use.

Reply:
I have no real issue with getting this ticket merged and raising a new one to update this URL when we get it. Bc I'm happy that the actual copy implementation/dev work is done

Reply:
I also could well be wrong!

Reply:
No worries, I thought i'd just check as it saves time but it seems it might take time to find out. I'll merge for now and pose the question and write a ticket if it needs to be updated